examples/src/llms/arcjet.ts

```diff
@@ -0,0 +1,42 @@
+import {
+  ArcjetRedact
+} from "@langchain/community/llms/arcjet";
+import { OpenAI } from "@langchain/openai";
+
+// Create an instance of another LLM for Arcjet to wrap
+const openai = new OpenAI({
```

Comment:
We probably will want this to work with chat models - most state of the art models are chat models rather than LLMs.

Reply:
We are planning on adding a second integration for chat models, I'm planning on adding that in a second PR though since it is a separate piece of work.